frontend/app/routes/$lang+/_public+/apply+/$id+/applicant-information.tsx

```diff
@@ -50,14 +55,25 @@ export async function loader({ request, params }: LoaderFunctionArgs) {
 export async function action({ request, params }: ActionFunctionArgs) {
   const applyFlow = getApplyFlow();
   const { id } = await applyFlow.loadState({ request, params });
+  const t = await getFixedT(request, handle.i18nNamespaces);
+
+  /**
+   * Schema applicant information.
+   */
+  const applicantInformationSchema: z.ZodType<ApplicantInformationState> = z.object({
+    socialInsuranceNumber: z.string().refine(isValidSin, { message: t('apply:applicant-information.error-message.valid-sin') }),
+    firstName: z.string().min(1, { message: t('apply:applicant-information.error-message.first-name') }),
+    lastName: z.string().min(1, { message: t('apply:applicant-information.error-message.last-name') }),
+    maritalStatus: z.string({ required_error: t('apply:applicant-information.error-message.marital-status') }),
+  });
 
   const formData = Object.fromEntries(await request.formData());
-  const parsedDataResult = applyFlow.applicantInformationSchema.safeParse(formData);
+  const parsedDataResult = applicantInformationSchema.safeParse(formData);
```

Comment:
Use [FormData: get() method](https://developer.mozilla.org/en-US/docs/Web/API/FormData/get) instead of using `Object.fromEntries`. We've started to use `.get()` to have more control on the data object that will be validated. As an example if you want to have all the checked values from a list of checboxes then we will use `.getAll()` that returns an arrays of values.

Here's an example how I did it.
https://github.com/DTS-STN/canadian-dental-care-plan/commit/5a6626edeb42a2713331650dbeb6e0eaaf3ae645#diff-34f5d4f6139e9c7445d3f867cb14241817fec4f7c0044f3026b88bdc604a9d22R67